vagrant-kubernetes-istio/install_prereqs_debian.sh

```diff
@@ -41,8 +42,26 @@ if [ $? -ne 0 ]; then
       exit 1
     fi
     sudo apt-get install -f
+    # Check for more recent version and update
```

Comment:
not sure i follow...

why dpkg and then api-get?

Reply:
Because we have seen that for linux install if virtual box is not present and install fails because of missing dependencies, api-get doesn't work properly.. so thus doing it after virtualbox is installed